js/src/Components/Widgets/CkeditorWidget.js

```diff
@@ -251,6 +251,10 @@ export class CkeditorWidget extends Component {
             </div>
           </div>
         </div>
+
+        {this.props.settings.help.length > 1 &&
+          <FormHelperText dangerouslySetInnerHTML={{__html: this.props.settings.help}}/>
```

Comment:
This works.

Reply:
Excellent, thanks.  I was trying to find a convenient way to render it as HTML, and I couldn't find a simple one without an additional library.